gwcli/storage.py

```diff
@@ -750,6 +805,9 @@ def ui_command_resize(self, size=None):
 
         self.resize(size)
 
+    def ui_command_reconfigure(self, attribute=None, value=''):
+        self.reconfigure(attribute, value)
```

Comment:
I can run "reconfigure" here and it doesn't throw an error for not specifying an attribute / value:

> /disks/rbd.lun0> reconfigure 
> Issuing reconfigure request: attribute=None, value=
> ok
>

Reply:
Thanks for testing this. I moved the validations to `Disks::reconfigure` method.